ldap/servers/slapd/slapi-private.h

```diff
@@ -1464,6 +1464,17 @@ void slapi_pblock_set_operation_notes(Slapi_PBlock *pb, uint32_t opnotes);
 void slapi_pblock_set_flag_operation_notes(Slapi_PBlock *pb, uint32_t opflag);
 void slapi_pblock_set_result_text_if_empty(Slapi_PBlock *pb, char *text);
 
+/* task warnings */
+typedef enum task_warning_t{
+    WRN_UPGRD_DNFMT_ALL = (1 << 0),
```

Comment:
I don't think these should be abbreviated, should be WARN_UPGRADE_DN_FORMAT etc. Helps future readers know what's going on :)